Lib9c/Model/CollectionMap.cs

```diff
@@ -53,7 +53,9 @@ public void Clear()
 
         public bool Contains(KeyValuePair<int, int> item)
         {
+#pragma warning disable LAA1002
             return _dictionary.Contains(item);
+#pragma warning restore LAA1002
```

Comment:
:+1: I am going to fix this false alarm: <https://github.com/planetarium/libplanet/issues/1048>.